doc/conf.py

```diff
@@ -189,143 +217,154 @@
 
 # If false, no module index is generated.
-#html_use_modindex = True
+# html_use_modindex = True
 
 # If false, no index is generated.
-#html_use_index = True
+# html_use_index = True
 
 # If true, the index is split into individual pages for each letter.
-#html_split_index = False
+# html_split_index = False
 
 # If true, links to the reST sources are added to the pages.
 html_show_sourcelink = False
 
 # If true, an OpenSearch description file will be output, and all pages will
 # contain a <link> tag referring to it.  The value of this option must be the
 # base URL from which the finished HTML is served.
-#html_use_opensearch = ''
+# html_use_opensearch = ''
 
 # If nonempty, this is the file name suffix for HTML files (e.g. ".xhtml").
-#html_file_suffix = ''
+# html_file_suffix = ''
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'nipypedoc'
+htmlhelp_basename = "nipypedoc"
 
 
 # -- Options for LaTeX output --------------------------------------------------
 
 # The paper size ('letter' or 'a4').
-#latex_paper_size = 'letter'
+# latex_paper_size = 'letter'
 
 # The font size ('10pt', '11pt' or '12pt').
-#latex_font_size = '10pt'
+# latex_font_size = '10pt'
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
-    ('interfaces', 'interfaces.tex', 'Nipype Interfaces Documentation',
-     'Neuroimaging in Python team', 'manual'),
+    (
+        "interfaces",
+        "interfaces.tex",
+        "Nipype Interfaces Documentation",
+        "Neuroimaging in Python team",
+        "manual",
+    ),
     # ('developers', 'developers.tex', 'Nipype API',
     #  'Neuroimaging in Python team', 'manual'),
-    ('examples', 'examples.tex', 'Nipype Examples',
-     'Neuroimaging in Python team', 'manual'),
+    (
+        "examples",
+        "examples.tex",
+        "Nipype Examples",
+        "Neuroimaging in Python team",
+        "manual",
+    ),
 ]
 
 # The name of an image file (relative to this directory) to place at the top of
 # the title page.
-#latex_logo = None
+# latex_logo = None
 
 # For "manual" documents, if this is true, then toplevel headings are parts,
 # not chapters.
-#latex_use_parts = False
+# latex_use_parts = False
 
 # Additional stuff for the LaTeX preamble.
-#latex_preamble = ''
+# latex_preamble = ''
 
 # Documents to append as an appendix to all manuals.
-#latex_appendices = []
+# latex_appendices = []
 
 # If false, no module index is generated.
-#latex_use_modindex = True
+# latex_use_modindex = True
 
 # -- apidoc extension configuration ------------------------------------------
-apidoc_module_dir = '../nipype'
-apidoc_output_dir = 'api/generated'
+apidoc_module_dir = "../nipype"
+apidoc_output_dir = "api/generated"
 apidoc_excluded_paths = [
-    '*/tests/*', 'tests/*',
-    'external/*',
-    'fixes/*',
-    'scripts/*',
-    'sphinxext/*',
```

Comment:
I removed this line, and then `black` had to have this file.

We're now at least exercising the code in doc builds again, even if it won't show up in coverage. I'm going to merge.